README.md

```diff
@@ -38,12 +38,24 @@ Create a new Markdown file in the group's directory, for example, `content/event
 
 ## For Developers
 
+### Requirements
+
+To run and contribute to the project, you will need to install the following requirements:
+
+1. [NodeJS](https://nodejs.org/en/download)
+2. [Yarn](https://classic.yarnpkg.com/lang/en/docs/install/#mac-stable)
+3. [Hugo](https://gohugo.io/installation/)
+
 ### Building the Calendar
 
 Install the dependencies with `yarn` then run `yarn build` to generate the calendar.
 
 ### Running the Hugo Server
 
+Before running the server, you need to install all required modules by running:
+
+    yarn install
+
 You need to have `hugo` installed on your machine. To run the server, execute:
```

Comment:
Thanks for making the changes.

We could probably remove this first sentence "You need to have `hugo` installed on your machine" as it's now redundant given your new Requirements block.

Reply:
That's true! Let me do that real quick!

Reply:
@zidhuss could you please check now?